Sources/App/Models/License.swift

```diff
@@ -112,20 +112,26 @@ enum License: String, Codable, Equatable {
         }
     }
 
-    var isCompatibleWithAppStore: Bool {
+    var licenseKind: Kind {
         switch self {
+            case .other,
+                 .none: return .noneOrUnknown
             case .agpl_3_0,
                  .gpl,
                  .gpl_2_0,
                  .gpl_3_0,
                  .lgpl,
                  .lgpl_2_1,
-                 .lgpl_3_0: return false
-            case .other,
-                 .none: return false
-            default: return true
+                 .lgpl_3_0: return .incompatibleWithAppStore
+            default: return .compatibleWithAppStore
         }
     }
+
+    enum Kind: String {
```

Comment:
Feels a bit weird having a type called `Kind`, but `License.LicenseKind` also felt weird.

Reply:
We could use ``enum `Type`: String {``

Reply:
I can fix that up if you're busy with the metada threads, just le me know :)